man/systemd.service.xml

```diff
@@ -1331,6 +1331,9 @@ WantedBy=multi-user.target</programlisting>
       <varname>ExecStart=</varname> specified. They will be executed
       in order until either they are all successful or one of them
       fails.</para>
+      <para><varname>Restart=</varname> can accept <option>on-failure</option>
+        <option>on-abnormal</option> <option>on-watchdog</option> <option>on-abort</option>.
+        If multiple <varname>ExecStart=</varname> are specified all will be run again.
```

Comment:
Please match the indentation of the surrounding text...

This seems to have invalid syntax:
```
FAILED: man/systemd.service.5 
/usr/bin/xsltproc -o man/systemd.service.5 --nonet --xinclude --maxdepth 9000 --stringparam man.output.quietly 1 --stringparam funcsynopsis.style ansi --stringparam man.authors.section.enabled 0 --stringparam man.copyright.section.enabled 0 --stringparam systemd.version 243 --path /tmp/autopkgtest-lxc.1fsh4vwd/downtmp/build.Soy/src/build-deb/man:/tmp/autopkgtest-lxc.1fsh4vwd/downtmp/build.Soy/src/man ../man/custom-man.xsl ../man/systemd.service.xml
../man/systemd.service.xml:1337: parser error : Opening and ending tag mismatch: para line 1334 and example
    </example>
              ^
../man/systemd.service.xml:1509: parser error : Opening and ending tag mismatch: example line 1300 and refsect1
  </refsect1>
             ^
../man/systemd.service.xml:1526: parser error : Opening and ending tag mismatch: refsect1 line 1254 and refentry
</refentry>
           ^
../man/systemd.service.xml:1527: parser error : Premature end of data in tag refentry line 6
^
```

You can use `build/man/man systemd.service` to build and view the man page.